docs/learn/learn-account-balances.md

```diff
@@ -10,19 +10,12 @@ remain frozen and unused due to an on-chain requirement.
 !!!info "A more efficient distribution of account balance types"
     Soon, pallets on Polkadot SDK will be implementing the _fungible_ trait (see [the tracking issue](https://github.com/paritytech/polkadot-sdk/issues/226) for more info). This new logic will allow for more efficient use of your account balance. Specifically, the [fungible trait](https://paritytech.github.io/polkadot-sdk/master/frame_support/traits/tokens/fungible/index.html) will allow using the `free` balance for on-chain activity like setting proxies and identities.
 
-There are 5 types of account balances:
-
-- **Free** is the balance that can be used for on-chain activity like staking, participating in
-  governance etc. but is not necessarily spendable (or transferrable)
-- **Frozen** is the free balance locked for [staking](./learn-staking.md),
-  [governance](./learn-polkadot-opengov.md), and [vesting](./learn-transactions.md#vested-transfers)
-  (also called locked balance)
-- **On hold** is used for [identities](./learn-identity.md), [proxies](./learn-proxies.md),
-  [OpenGov preimages and deposits](./learn-guides-polkadot-opengov.md#claiming-opengov-deposits),
-  and it is no longer free (also called reserved balance)
-- **Spendable** is the free balance that can be spent
-- **Untouchable** is the portion of the free balance that cannot be moved (i.e., not spendable) but
-  can still be used for on-chain activity
+There are 4 types of account balances:
+
+- **Free** is the balance that can be used for any on-chain activity ([staking](./learn-staking.md), [governance](./learn-polkadot-opengov.md), deposits) as long as your total balance (free + reserved) remains above the maximum of frozen balance and existential deposit.
+- **Frozen** (also called locks) is a balance that overlaps across pallets. Example: If governance freezes 100 DOT and vesting freezes 120 DOT, total frozen = 120 DOT (not 220 DOT).
+- **Reserved** (also called holds) is the balance removed from free and doesn't overlap. Used by nomination pools, [staking](./learn-staking.md), etc. Can still be used for governance voting but not for transfers or fees.
+- **Spendable** is the portion of free balance available for transaction fees and creating new holds.
 
 The spendable balance is calculated as follows:
```

Comment:
This needs to be updated as part of this PR. E.g. `on_hold` is no longer defined. Does the formula need an update? @Ank4n

Reply:
I think this is sadly not correct, since a reserved balance creates a provider ref which makes the ED spendable. I will check again how it is exactly, but more complicated than the formula currently.

Reply:
IMO it should be something like this. It is also not 100% correct though, see the NOTE. So not sure how to *easily* explain this. In practice its not easy.

```rust
untouchable = frozen - on_hold.min(frozen);

// NOTE: The `(on_hold > 0 || frozen > 0)` is an *approximation* for
// `(consumers != 0 && providers <= 1)`, which is the condition that actually matters.
if free > 0 && (on_hold > 0 || frozen > 0) {
	untouchable = untouchable.max(ed);
}

spendable = free - untouchable.min(free);
```

Reply:
We have a [test case here](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/staking-async/src/tests/bonding.rs#L63) demonstrating that when a hold exists, the ED is not spendable.

Here’s the `AccountInfo` for 11 from that test (not printed in the test output, but retrieved by running it locally):

```
AccountInfo {
    nonce: 0,
    consumers: 1,
    providers: 1,
    sufficients: 0,
    data: AccountData {
        free: 1,
        reserved: 1000,
        frozen: 0,
        flags: ...
    }
}
```